cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)
+    description = serializers.CharField(allow_blank=True)
+    duration = serializers.IntegerField(required=True)
```

Comment:
Consider using `PositiveIntegerField` for the `duration` field in the serializer to match the suggested change in the model, ensuring only positive values are serialized.